Source/Fuse.Maps/MapView.uno

```diff
@@ -241,6 +238,10 @@ namespace Fuse.Controls
 		bool _willUpdateCameraNextFrame;
 		internal void UpdateCameraNextFrame()
 		{
+			if defined(!MOBILE) {
+			        map.UpdateMap();
+			        return;
+			}
```

Comment:
This fixed the .NET error:
```diff
diff --git a/Source/Fuse.Maps/MapView.uno b/Source/Fuse.Maps/MapView.uno
index 4fcc6cc..6643e58 100644
--- a/Source/Fuse.Maps/MapView.uno
+++ b/Source/Fuse.Maps/MapView.uno
@@ -238,13 +238,18 @@ namespace Fuse.Controls
 		bool _willUpdateCameraNextFrame;
 		internal void UpdateCameraNextFrame()
 		{
-			if defined(!MOBILE) {
-			        map.UpdateMap();
-			        return;
+			if defined(!MOBILE)
+			{
+				map.UpdateMap();
+			}
+			else
+			{
+				if(!MapIsReady || _willUpdateCameraNextFrame)
+					return;
+
+				UpdateManager.PerformNextFrame(ApplyCameraState, UpdateStage.Primary);
+				_willUpdateCameraNextFrame = true;
 			}
-			if(!MapIsReady || _willUpdateCameraNextFrame) return;
-			UpdateManager.PerformNextFrame(ApplyCameraState, UpdateStage.Primary);
-			_willUpdateCameraNextFrame = true;
 		}
 
 		void ApplyCameraState()
```

The problem here, is that .NET doesn't like it when we have dead code after an unconditional return. Uno is not so clever at optimizing this away.